diktat-api/src/main/kotlin/com/saveourtool/diktat/DiktatRunnerArguments.kt

```diff
@@ -21,7 +21,7 @@ import kotlin.io.path.inputStream
  */
 data class DiktatRunnerArguments(
     val configInputStream: InputStream,
-    val sourceRootDir: Path,
+    val sourceRootDir: Path?,
```

Comment:
why nullable?

Reply:
it needs to support a full path, when `sourceRootDir` is not provided, diktat reporter doesn't resolve relative path for processed files